modules/ephemeris/src/test/scala/gem/horizons/HorizonsEphemerisQueryPagingPropSpec.scala

```diff
@@ -54,15 +54,27 @@ class HorizonsEphemerisQueryPagingPropSpec extends PropSpec with PropertyChecks
     }
   }
 
-  property("last element should be less than one step size away from end") {
+  property("last element should be less than two step sizes away from end") {
     forAll { (t: TestCase) =>
       t.paging.lastOption match {
         case None    => fail
-        case Some(q) => q.endTime should (be >= t.end and be <= (t.end + t.step))
+        case Some(q) => q.endTime should (be >= t.end and be < (t.end + t.step * 2))
       }
     }
   }
 
+  property("explicitly handle case where last page would have one element using MaxElement-sized pages") {
```

Comment:
nice

Reply:
This was a randomly generated test case that produced the issue in the first place.  It's unlikely to come up very often randomly so it seemed like a good idea to leave it as a test case permanently.